docs/pages/includes/database-access/gcp-public-address-note.mdx

```diff
@@ -0,0 +1,10 @@
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  The Teleport Database service can connect to public or private addresses.
+  To use a private address the Teleport Database service must be able to connect via its
+  network connections, typically by being in the same Zone and Region as the GCP service.
```

Comment:
What does "must be able to connect via its network connections" mean?

Reply:
Given the address of the database it needs to be able to connect from whatever location the Teleport Database Service is running.  That could be within the same Zone and Region or using peer-to-peer connection.

Reply:
I would reconsider the wording of this tip, since for me, it sounds fairly commonsensical, and only the "See Google Cloud documentation" note at the end implies that there is a special situation users need to be aware of.

Maybe start with a sentence that states the counterintuitive or surprising behavior, then use the rest of the paragraph to provide further explanation.